csrankings-m.csv

```diff
@@ -1876,6 +1876,7 @@ Michael Zakharyaschev,Birkbeck University of London,https://www.dcs.bbk.ac.uk/ab
 Michael Zapp,University of Manitoba,http://cs.umanitoba.ca/people,pN2BYuIAAAAJ
 Michael Zbyszynski,Goldsmiths University of London,https://www.gold.ac.uk/computing/people/zbyszynski-michael,OAnw6WwAAAAJ
 Michael Zink,University of Massachusetts Amherst,http://www.ecs.umass.edu/ece/zink,iVck6WIAAAAJ
+Michael Zuzak,Rochester Institute of Technology,https://mzuzak.github.io,dESHYYcAAAAJ

```

Comment:
CE, not CS. Please remove or justify how they meet the inclusion criteria.

Reply:
totally right, see below